files/galaxy/config/object_store_conf.xml

```diff
@@ -41,11 +41,16 @@
             <extra_dir type="temp" path="/data/dnb02/galaxy_db/tmp"/>
             <extra_dir type="job_work" path="/data/dnb03/galaxy_db/job_working_directory"/>
         </backend>
-        <backend id="files10" type="disk" weight="1" store_by="uuid">
+        <backend id="files10" type="disk" weight="0" store_by="uuid">
             <files_dir path="/data/dnb03/galaxy_db/files"/>
             <extra_dir type="temp" path="/data/dnb02/galaxy_db/tmp"/>
             <extra_dir type="job_work" path="/data/dnb03/galaxy_db/job_working_directory"/>
         </backend>
+        <backend id="files11" type="disk" weight="1" store_by="uuid">
```

Comment:
what about using it only to 10% and see how it goes? Increasing tomorrow etc ...

Reply:
I really like the idea!